project/BRANCHES-AND-TAGS.md

```diff
@@ -0,0 +1,35 @@
+Branches and tags
+=================
+
+Note: details of the release process for the Engine are documented in the
+[RELEASE-CHECKLIST](https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md).
+
+# Branches
+
+Docker projects should normally have only three living branches at all time, including the regular
+`master` branch:
+
+## `docs` branch
+
+The `docs` branch support documentation updates between product releases. This branch allow us to
+decouple documentation releases from product releases.
+
+## `release` branch
```

Comment:
We don't only maintain the last release: we do security patches on version N and N-1.